Fix README collisions (#140)
github/issue-metrics: README.md

[![CodeQL](https://github.com/github/issue-metrics/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/github/issue-metrics/actions/workflows/codeql-analysis.yml) [![Docker Image CI](https://github.com/github/issue-metrics/actions/workflows/docker-image.yml/badge.svg)](https://github.com/github/issue-metrics/actions/workflows/docker-image.yml) [![Python package](https://github.com/github/issue-metrics/actions/workflows/python-package.yml/badge.svg)](https://github.com/github/issue-metrics/actions/workflows/python-package.yml)

This is a GitHub Action that searches for issues/pull requests/discussions in a repository, measures several metrics, and generates a report in form of a GitHub issue.
The issues/pull requests/discussions to search for can be filtered by using a search query.

| Metric | Description |
|--------|-------------|
|Time to First Response | The duration from creation to the initial comment or review.|
|Time to Close | The period from creation to closure.|
|Time to Answer (Discussions Only) | The time from creation to an answer.|

> |Time in Label | The time between when a label has a specific label applied to an issue/pull request/discussion and when it is removed. This requires the `LABELS_TO_MEASURE` env variable to be set.|

Reviewer: ```suggestion
|Time in Label | The duration from label application to removal, requires `LABELS_TO_MEASURE` env variable.|
```

I had actually only formatted the variable here.

So if you want to keep the shorter description from the other PR, you might have to adapt this as shown above.
